src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -171,6 +171,22 @@ public void TestNormalizeStatement1()
             TestNormalizeStatement("Func<string, int> f = blah;", "Func<string, int> f = blah;");
         }
 
+        [Theory]
+        [InlineData("int*p;", "int* p;")]
+        [InlineData("int *p;", "int* p;")]
+        [InlineData("int*p1,p2;", "int* p1, p2;")]
+        [InlineData("int *p1, p2;", "int* p1, p2;")]
+        [InlineData("int**p;", "int** p;")]
+        [InlineData("int **p;", "int** p;")]
+        [InlineData("int**p1,p2;", "int** p1, p2;")]
+        [InlineData("int **p1, p2;", "int** p1, p2;")]
+        [InlineData("delegate * <int, void> p1;", "delegate *<int, void> p1;")]
```

Comment:
@jcouv This is the current outcome of the normalization. Can you confirm whether it's the correct one or file an issue to track fixing it?

Reply:
I appreciate. Let's file a tracking issue (doesn't block this PR).
I think the expected format would be `delegate*<int, void> p1;` with no space before or after the asterisk. Tagging @333fred to confirm.

Reply:
Correct, it should have no spaces.

Reply:
Filed tracking issue https://github.com/dotnet/roslyn/issues/50664

Reply:
Thanks @jcouv @333fred 

```suggestion
```